index.js

```diff
@@ -37,30 +32,19 @@ module.exports = {
         ' having trouble registering itself to the parent application.');
     }
 
-    var pathBase = this.project.addonPackages['@html-next/flexi-default-styles'].path;
-    compileScssVariables(path.join(pathBase, 'addon/styles'), this.flexiConfig());
-
     this.app = app;
     return app;
   },
 
-  isDevelopingAddon: function() {
-    return false;
-  },
-
-  _flexiConfig: null,
-  flexiConfig: function() {
-    if (!this._flexiConfig) {
-      this._flexiConfig = getValidatedFlexiConfig(this.project.root);
-    }
-
-    return this._flexiConfig;
+  treeForAddonStyles(tree) {
+    return mergeTrees([
+      tree,
```

Comment:
This appears to work correctly, though it isn't the same [approach used in the original PR](https://github.com/html-next/flexi-default-styles/pull/7/files#diff-168726dbe96b3ce427e7fedce31bb0bcR49).